sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIOIT.java

```diff
@@ -19,55 +19,67 @@
 
 import static org.apache.beam.sdk.io.gcp.healthcare.HL7v2IOTestUtil.HEALTHCARE_DATASET_TEMPLATE;
 
+import com.google.api.services.healthcare.v1.model.DeidentifyConfig;
 import java.io.IOException;
 import java.net.URISyntaxException;
-import org.apache.beam.sdk.PipelineResult;
-import org.apache.beam.sdk.testing.PAssert;
+import java.security.SecureRandom;
 import org.apache.beam.sdk.testing.TestPipeline;
-import org.apache.beam.sdk.transforms.Create;
 import org.junit.After;
-import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 
 @SuppressWarnings({"nullness", "uninitialized"})
-public class DicomIOReadIT {
-  private static final String TEST_FILE_PATH = "src/test/resources/DICOM/testDicomFile.dcm";
-  private static final String TEST_FILE_STUDY_ID = "study_000000000";
+public class DicomIOIT {
```

Comment:
The test has been rewritten. Is there a chance to remove these SuppressWarnings?